.rubocop.yml

```diff
@@ -776,12 +777,21 @@ Rails/HttpPositionalArguments:
     - spec/**/*
     - test/**/*
 
+Rails/I18nLocaleAssignment:
+  Enabled: false
+
+Rails/NegateInclude:
+  Enabled: false
```

Comment:
This would force `!x.include?(:y)` to be `x.exclude?(:y)` and I decided that similar to `unless` I think we should trust the humans to make the call on this one for readability